src/components/BetaInfoBox.jsx

```diff
@@ -0,0 +1,43 @@
+/* global _ */
+import React, { useState } from 'react';
+import Alert from './ui/Alert';
+import { Button } from './ui';
+import { get, set } from 'src/adapters/store';
+import { parseQueryString } from '../libs/url_utils';
+
+const initialQueryParams = parseQueryString(window.location.search);
+
+const BetaInfoBox = () => {
+  const [closed, setClosed] = useState(get('beta_popup_closed'));
+  const [client] = useState(initialQueryParams['client']);
```

Comment:
Just a remark for your next functional components: as you don't allow it to be changed from the component itself (there is no setter), it doesn't need to use a state for that. You could just extract the value as a normal const in the function.